app/src/main/java/org/mozilla/fenix/search/toolbar/SearchSelectorToolbarAction.kt

```diff
@@ -52,7 +52,7 @@ class SearchSelectorToolbarAction(
                 }
 
                 UnifiedSearch.searchMenuTapped.record(NoExtras())
-                menu.menuController.show(anchor = it, orientation = orientation)
+                menu.menuController.show(anchor = it, orientation = orientation, forceOrientation = true)
```

Comment:
For both of these changes, is this a behaviour we are ready to ship by default or should i still be behind the unified search feature flag?

Reply:
This will still be behind the unified search flag.  There are some more that needs to be completed before removing the unified search feature flag.